jsapp/js/components/common/tooltip.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export interface TooltipProps {
+  text: string;
+  ariaLabel: string;
+  alignment?: 'right' | 'left' | '';
+}
+
+/**
+ * Tooltip component that is necessary for managing dynamic content and
+ * accessibility features, such as readability. While we have the tooltip.scss
+ * component, that provides styling but not other functionaltiy such as
+ * allowing tooltips to work on disabled buttons.
+ *
+ * @param {string} text - Content of the tooltip.
+ * @param {string} ariaLabel - Accessible label for screen readers.
+ * @param {'right' | 'left' | ''} alignment - Position of the tooltip (centered
+ * if undefined or set to an empty string).
+ * @param {React.ReactNode} children - Button that triggers the tooltip when
+ * hovered.
+ */
```

Comment:
This is great, just insted of all those `@param`s, we can describe each of parameters inside `TooltipsProps` interface - adding `/** comment */` above each of them. See `textBox.tsx` for example code. Sorry for not being clearer.

That double asterisk comment is being understood by text editors and being displayed, for example in Sublime Text:
<img width="596" alt="Screenshot 2023-12-11 at 13 29 19" src="https://github.com/kobotoolbox/kpi/assets/2521888/29f657c0-26a4-4f30-a410-913789aca812">